vllm/transformers_utils/config.py

```diff
@@ -70,7 +72,22 @@ def file_or_path_exists(model: Union[str, Path], config_name, revision,
     if Path(model).exists():
         return (Path(model) / config_name).is_file()
 
-    return file_exists(model, HF_CONFIG_NAME, revision=revision, token=token)
```

Comment:
FYI I _think_ this was a bug before, since for mistral models we'd want to be looking for the mistral config name on the hub